docker-compose.yml

```diff
@@ -142,6 +142,9 @@ services:
   worker:
     << : *sentry_defaults
     command: run worker
+  post-process-forwarder:
+    << : *sentry_defaults
+    command: run post-process-forwarder --commit-batch-size 3
```

Comment:
You might have trouble with batch size of 3. I’d do 1 by default for onpremise.

Reply:
Tested locally, seems to work fine with 1 event. Also, if I change this, should I change it for `snuba-replacer` above?

Reply:
I’m not really sure. From my understanding this wouldn’t flush until the minimum batch size. This was specifically an issue with the consumers in sentry. In snuba, there’s another flag for a min batch duration. So you could say batch size of 1000, or 1 second. So even if you don’t get 1000 messages, it’ll flush after 1 second. Last I knew about post-process-forwarder, it didn’t have the time component. So if you have a size greater than 1, it’s possible you don’t fill the batch, and it’ll never flush. This may have changed since though.

Reply:
Just checked the code and still no time-component so I'm reducing this to `1` as you suggested. We may wanna add the time component soon.

Reply:
I agree. I’ve been asking for it for months. :)